test/browser/spec.js

```diff
@@ -124,4 +124,49 @@ describe('Component spec', () => {
 			expect(WithDefaultProps.prototype.getDefaultProps).to.be.calledOnce;
 		});
 	});
+
+	describe('forceUpdate', () => {
+		it('should force a rerender', () => {
+			let forceUpdate;
+			class ForceUpdateComponent extends Component {
+				componentWillUpdate() {}
+				componentDidMount() {
+					forceUpdate = () => this.forceUpdate();
+				}
+				render() {
+					return <div />;
+				}
+			}
+			sinon.spy(ForceUpdateComponent.prototype, 'componentWillUpdate');
+			sinon.spy(ForceUpdateComponent.prototype, 'forceUpdate');
+			render(<ForceUpdateComponent />, scratch);
+			expect(ForceUpdateComponent.prototype.componentWillUpdate).not.to.have.been.called;
+
+			forceUpdate();
+
+			expect(ForceUpdateComponent.prototype.componentWillUpdate).to.have.been.called;
+			expect(ForceUpdateComponent.prototype.forceUpdate).to.have.been.called;
+		});
+
+		it('should add callback to renderCallbacks', () => {
```

Comment:
`renderCallbacks` is an implementation detail which we should avoid in the description. What the test actually _does_, checking that `forceUpdate` invokes its callback, is fine though.